x/concentrated-liquidity/pool_test.go

```diff
@@ -1,1071 +1,95 @@
 package concentrated_liquidity_test
 
 import (
+	"fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	cl "github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity"
-	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/internal/math"
+	clmodel "github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/model"
 	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/types"
+	swaproutertypes "github.com/osmosis-labs/osmosis/v13/x/swaprouter/types"
 )
 
-func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
```

Comment:
Note to reviewer: the removed tests here were all moved to swaps_test.go since that is where they belong